embed/templates/scripts/run_grafana.sh.tpl

```diff
@@ -6,6 +6,15 @@ set -e
 DEPLOY_DIR={{.DeployDir}}
 cd "${DEPLOY_DIR}" || exit 1
 
+# try to compatible with tidb-lightning's dashboard,
```

Comment:
We need set this in https://github.com/pingcap/tidb-lightning/blob/master/metrics/grafana/lightning.json instead of set it with script in tiup-cluster. Because there are other tools depends on it, too.

Reply:
Good idea, the _lightning.json_ needs to be modified,  and for the existing offline tarballs, we need to compatible and fix it.